README.md

```diff
@@ -228,36 +208,3 @@ Into AWS lambda:
 import logging
 logging.getLogger("awswrangler").setLevel(logging.DEBUG)
 ```
-
-## Who uses AWS SDK for pandas?
```

Comment:
Are we getting rid of this entirely? If so, we should get rid of the [issue template](https://github.com/aws/aws-sdk-pandas/blob/main/.github/ISSUE_TEMPLATE/support-data-wrangler.md) as well.

Reply:
This is up for discussion. I think the library is popular enough these days that we don't need public endorsing that can be outdated. But yes good point on the issue template, will delete it